### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by woomurf**
Added docs for blank part

### CODE REVIEW

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

File: src/nft.ts [hunk 1]
```diff
@@ -26,6 +26,11 @@ import {
 } from './types';
 
 export default class Nft extends AinftBase {
+  /**
+   * Add nfy symbol. You can add nft to reference in your factory activity.
+   * @param param0
```
Comment (yoojinko) [inline src/nft.ts:31]: It appears that the object parameters are currently labeled as param0. 
How about clarifying object types instead of param0?

Comment (woomurf) [reply src/nft.ts]: Updated. Thanks!

**Review by yoojinko — APPROVED** [review]
LGTM, Thanks!

**Comment by woomurf** [comment]
Thanks for the review!

**Pull request merged**